src/Bridge/Doctrine/EventListener/PurgeHttpCacheListener.php

```diff
@@ -40,13 +40,19 @@ final class PurgeHttpCacheListener
     private $resourceClassResolver;
     private $propertyAccessor;
     private $tags = [];
+    private $xKeyPurger;
+    private $xkey;
```

Comment:
It does no seem to be used.

Reply:
It is used in the postFlush method L112-118 to determine if tags need to be purged using either/both xkey or/and Cache-Tags

Reply:
Renaming it to `$xkeyEnabled` would be clearer then.

Reply:
Done.